openstates/in/bills.py

```diff
@@ -45,198 +45,185 @@ class INBillScraper(BillScraper):
+                bill.add_sponsor("primary",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="sponsor")
+
+            for s in bill_json["cosponsors"]:
+                bill.add_sponsor("cosponsor",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="cosponsor")
+
+            #actions
+            action_link = bill_json["actions"]["link"]
+            api_source = api_base_url + action_link
+            actions = client.get("bill_actions",session=session,bill_id=bill_id.lower())
+            for a in actions["items"]:
+                action_desc = a["description"]
+                chamber = "lower" if a["chamber"]["name"].lower() == "house" else "upper"
+                date = a["date"]
+                date = datetime.datetime.strptime(date,"%Y-%m-%dT%H:%M:%S")
+                
+                action_type = []
+                d = action_desc.lower()
+                committee = None
+                if "first reading" in d:
+                    action_type.append("bill:reading:1")
+
+                if ("second reading" in d
+                    or "reread second time"):
+                    action_type.append("bill:reading:2")
+
+                if "third reading" in d:
+                    action_type.append("bill:reading:3")
+                    if "passed" in d:
+                        action_type.append("bill:passed")
+                    if "failed" in d:
+                        action_type.append("bill:failed")
+
+                if ("referred" in d and "committee on" in d
+                    or "reassigned" in d and "committee on" in d):
+                    committee = d.split("committee on")[-1].strip()
+                    action_type.append("committee:referred")
+
+
+                if "committee report" in d:
+                    if "pass" in d:
+                        action_type.append("committee:passed")
+                    if "fail" in d:
+                        action_type.append("committee:failed")
+
+                if "amendment" in d:
+                    if "pass" in d or "prevail" in d:
+                        action_type.append("amendment:passed")
+                    if "fail" or "out of order" in d:
+                        action_type.append("amendment:failed")
+                    if "withdraw" in d:
+                        action_type.append("amendment:withdrawn")
+
+
+                #blacklisted actions - don't correspond to anything we know
+                if ("authored" in d
+                    or "referred to the senate" in d
+                    or "referred to the house" in d
+                    or "sponsor" in d
+                    or "coauthor" in d
+                    or ("rule" in d and "suspended" in d)):
+                    #not really actions we track
                     continue
```

Comment:
`pass` instead of `continue`? I'm thinking of situations where one of these blacklisted actions _and_ another action type appear in the same `d`; we'd want to still `add_action()` instead of skipping, right?